conandata.yml

```
@@ -87,6 +87,11 @@
                 src: "bin"
                 dst: "."
                 binary: "CuraEngine"
+            spdlog:
+                package: "spdlog"
+                src: "include"
```

This shouldn't work. The `include` directory of `spdlog` package contains the header files `*.h` not the shared library.

---

the `*.so` files are located in `.conan/data/spdlog/1.10.0/_/_/package/<hash>/lib/`

so it should probably be something like:
```suggestion
                src: "lib"
```

but this might give trouble on Windows because on Windows it will be probably a `dll` and stored in the `bin` folder